starfish/pipeline/filter/richardson_lucy_deconvolution.py

```diff
@@ -8,60 +8,65 @@
 from starfish.image import ImageStack
 from ._base import FilterAlgorithmBase
 from .util import gaussian_kernel
+from starfish.typing import Number
 
 
 class DeconvolvePSF(FilterAlgorithmBase):
 
     def __init__(
-            self, num_iter: int=15, sigma: float=2, clip: bool=False, verbose=False, **kwargs
-    ) -> None:
+            self, num_iter: int, sigma: Number, clip: bool=False, **kwargs) -> None:
         """Deconvolve a point spread function
 
-        Note that the default parameters are highly optimized for the MERFISH use case and that num_iter is a
-        very important parameter that requires careful optimization.
-
         Parameters
         ----------
         num_iter : int
-            number of iterations to run
-        sigma : float
+            number of iterations to run. Note that this is a very important parameter that requires
+            careful optimization
+        sigma : Number
             standard deviation of the gaussian kernel used to construct the point spread function
         clip : bool (default = False)
-            if True, pixel values below -1 and above 1 are clipped for skimage pipeline compatibility
-        verbose : bool
-            if True, report on the percentage completed during processing (default = False)
+            if True, pixel values below -1 and above 1 are clipped for skimage pipeline
+            compatibility
 
         """
         self.num_iter = num_iter
         self.sigma = sigma
         self.clip = clip
         self.kernel_size: int = int(2 * np.ceil(2 * sigma) + 1)
-        self.psf: np.ndarray = gaussian_kernel(shape=(self.kernel_size, self.kernel_size), sigma=sigma)
-        self.verbose = verbose
+        self.psf: np.ndarray = gaussian_kernel(shape=(self.kernel_size, self.kernel_size),
```

Comment:
prefer the no args on the first line if you wrap, i.e.,

```
self.psf: np.ndarray = gaussian_kernel(
    shape=(self.kernel_size, self.kernel_size), sigma=sigma)
```

Reply:
👍 I split it out over a few lines, one arg per line.